RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
Erf yeah, good point (that frankly sucks, the Azure SDK should do the mapping itself instead of forcing us to do it...). 

At this point, maybe it would be better to remove the 2.0 TFM?